lib/web/scripts/desktop/install-ad-cs.ps1

```diff
@@ -0,0 +1,5 @@
+$ErrorActionPreference = "Stop"
+
+Add-WindowsFeature Adcs-Cert-Authority -IncludeManagementTools
+Install-AdcsCertificationAuthority -CAType EnterpriseRootCA -Force
+Restart-Computer -Force
```

Comment:
Does it do what it says it does? 😄 I.e. will it actually reboot the machine where user runs this script without prompt? Is that expected UX?

Reply:
Yes! It reboots the machine without the prompt. I think that it is expected UX as you can't go to the next step without rebooting the machine. Also, these two first steps where we install AD/DS and CA services are usually performed on the machines that are used for evaluation of the Teleport desktop service or for developement and testing. In most cases for production use there is already configured Active Directory so we skip step 1 and step 2.

Reply:
Okay, I see. If that's the behavior we want and it's ok with @zmb3 and @xinding33 then sounds good. I would at least consider adding a confirmation prompt or some kind of message warning the user that their instance will be rebooted.

Reply:
I'm fine with it. The web UI does say "requires reboot"